src/lib.rs

```diff
@@ -72,14 +72,7 @@
     // Covered by other lints
     clippy::missing_panics_doc, // clippy::unwrap_used
 )]
-#![deny(
-    clippy::unwrap_used,
-    clippy::non_ascii_literal,
-    clippy::missing_errors_doc,
-    clippy::let_underscore_must_use,
-    clippy::unused_async,
-    unused
-)]
+#![warn(clippy::unwrap_used, clippy::non_ascii_literal, clippy::let_underscore_must_use, unused)]
```

Comment:
One final nit, you may want to combine this with the `#![warn(clippy::pedantic)]` up above.